### DISCUSSION TITLE
[Idea] Münzen sind Checkpoint abhängig

### DISCUSSION THREAD

**Discussion by KnolleShiroi** [FloSet/BeatPass#24] [Ideas]
Wenn eine Münze eingesammelt wird und man dann ein Leben verliert, wird man zum letzten Checkpoint zurückgesetzt und die Münze gilt als nicht aufgesammelt und muss erneut eingesammelt werden. Die Münze wird beim nächsten Checkpoint erst gespeichert.

**Comment by FloSet** [comment]
Dies ist nun integriert!
